### PR TITLE
Ensure raw tag has no arguments

### PR DESCRIPTION
**Pull request by pushrax**
We usually don't enforce the lack of arguments to zero-argument tags (e.g. break, continue, etc), but I think it's a good practise.

cc @gauravmc

### CODE REVIEW

**Comment by parkr** [comment]
:+1:

File: lib/liquid/tags/raw.rb [hunk 1]
```diff
@@ -1,7 +1,16 @@
 module Liquid
   class Raw < Block
+    Syntax = /\A\s*\z/om
```
Comment (gauravmc) [inline lib/liquid/tags/raw.rb]: Don't need `o`.. probably also `m`

**Comment by gauravmc** [comment]
:+1:

File: test/integration/tags/raw_tag_test.rb [hunk 2]
```diff
@@ -24,6 +24,7 @@ def test_open_tag_in_raw
   end
 
   def test_invalid_raw
-    assert_match_syntax_error /tag was never closed/, '{% raw } foo {% endraw %}'
```
Comment (fw42) [inline test/integration/tags/raw_tag_test.rb]: where did this test go? why did you delete it?

Comment (pushrax) [inline test/integration/tags/raw_tag_test.rb]: It ends up hitting the new case and getting the other syntax error, so I edited it to new line at that location.

Comment (fw42) [inline test/integration/tags/raw_tag_test.rb]: I would prefer to keep all three tests. It's not obvious to me that the new tests cover the exact same codepaths that the old test did.

Comment (pushrax) [inline test/integration/tags/raw_tag_test.rb]: Sure.

**Comment by fw42** [comment]
:+1:

**Comment by gauravmc** [comment]
Would need a gem bump after.. require this is Shopify: https://github.com/Shopify/shopify/pull/46102
